app/developers/templates/developers/collection_submit_confirmation.html

```diff
@@ -2,21 +2,20 @@
 {% extends "developers/access_grant_funding_base.html" %}
 
 {% block pageTitle %}
-  Collection submitted {{ collection_helper.name }} - MHCLG Funding Service
+  Submission submitted {{ submission_helper.name }} - MHCLG Funding Service
```

Comment:
Just for good measure i'll make a note that I'm happy to push for the non-domain specific "completed" language for now, I didn't have conviction earlier but it looks clear seeing it all in the round

The helper already uses langauage around the generic mechanism like `submission.is_completed` so lets line everything up with that language and use domain language (i.e competitive application for funding is submitted) when we come to it

"Submission completed"
`SUBMISSION_COMPLETED`

Reply:
I was definitely fine with "submitted" as a neutral enough term in the before times but really reaching here

Reply:
agree